lib/broadway_dashboard/nav_bar_component.ex

```diff
@@ -0,0 +1,164 @@
+defmodule BroadwayDashboard.NavBarComponent do
+  @moduledoc false
+  # most of code copy from Phoenix.LiveDashboard.NavBarComponent
+  # but this module support string for the value of nav
```

Comment:
Let's instead send a PR to NavBarComponent in LiveDashboard to actually normalize all navs to strings behind the scenes. :)

Reply:
OK